content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -15,68 +15,84 @@ further_reading:
       tag: 'GitHub'
       text: 'Examples of Custom Instrumentation'
 ---
+<div class="alert alert-warning">
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution in your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+</div>
+
 
 ## Compatibility
 
 - The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual Basic).
 
-- The .NET Tracer supports instrumentation on:
-  - .NET 5
-  - .NET Core 3.1
-  - .NET Core 2.1
+- The .NET Tracer library for Datadog is open-source. For more information, see the [tracer Github repository][1].
 
-- .NET Tracer supports the following processor architectures
-  - Windows x86 (`win-x86`)
-  - Windows x64 (`win-x64`)
-  - Linux x64 (`linux-x64`)
-  - Alpine Linux x64 (`linux-musl-x64`)
-  - Linux ARM64 (`linux-arm64`) Added in version 1.27.0, automatic instrumentation only supported on .NET 5.
+### Supported .NET Core runtimes
```

Comment:
The reason I left the Compatibility section was to provide space for the following lines: 

- The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual Basic).

- The .NET Tracer library for Datadog is open-source. For more information, see the tracer Github repository.

Otherwise they would not have a section header.